requirements/developer.txt

```diff
@@ -10,4 +10,5 @@ grpcio
 protobuf
 grpcio-tools
 transformers==4.11.0
-pygit2==1.6.1
```

Comment:
@msaroufim is installing "aspell" as mentioned in PR description still relevant?

Reply:
@msaroufim seems it's still required to have "aspell" should we add it to the requirements?

 Regarding the vocab provided for spell checking, I wonder if lowercase of some of he words needs to be added as well? for example config, [here](https://gist.github.com/HamidShojanazeri/7433ef4f99a12b84be843aa3eb914714#file-gistfile1-txt-L27) in doc/serve.md

Reply:
@HamidShojanazeri these should be fixed